rdkit/Chem/UnitTestRandomize.py

```diff
@@ -0,0 +1,47 @@
+#
+#  Copyright (C) 2017 Esben Jannik Bjerrum
+#
+#   @@ All Rights Reserved @@
+#  This file is part of the RDKit.
+#  The contents are covered by the terms of the BSD license
+#  which is included in the file license.txt, found at the root
+#  of the RDKit source tree.
+#
+from __future__ import print_function
+
+import unittest
+
+from rdkit import Chem
+from rdkit.Chem import Randomize
+
+
+def MolBlockToSmiles(mb):
+  return Chem.MolToSmiles(Chem.MolFromMolBlock(mb))
+
+class TestCase(unittest.TestCase):
+
+  def test1(self):
+    smiles = ["C[N+](C)(C)C", "CC(=O)[O-]", "[NH3+]CC(=O)[O-]",'[Cl-].[Cl-].[Mg+2]']
+    msg = "Could not randomize charged mol %s"
+    for smi in smiles:
+      mb = Chem.MolToMolBlock(Chem.MolFromSmiles(smi))
+      rmb = Randomize.RandomizeMolBlock(mb)
```

Comment:
While I don't see that it would cause a problem here, the use of random will create a _different_ test every time. You can seed the random module using e.g. `random.seed(0)`. This will create the same randomized mol block every time. 

To cover a few more cases, you could then also repeat the randomization for every molecule several times (10, 20, or so) as it's done in `test2`

It's also useful to test edge cases, MgCl2 is a good one as it only contains charged atoms. [O-][O-] would also be interesting as it will have the charged atom as the first and the last atom; [O-]C[O-] would also be good for the same reason.